tests/containers/SignIn.test.jsx

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+import { mount } from 'enzyme';
+import { MemoryRouter } from 'react-router-dom';
+import localStorage from 'mock-local-storage';
```

Comment:
`localStorage` is defined but never used.

Reply:
This is required to mock localStorage. Without it this `window.localStorage = global.localStorage` won't work